GeneratorInterface/RivetInterface/python/genParticles2HepMC_cfi.py

```diff
@@ -3,4 +3,6 @@
 genParticles2HepMC = cms.EDProducer("GenParticles2HepMCConverter",
     genParticles = cms.InputTag("genParticles"),
```

Comment:
would it make sense to add another commented line like
#     genParticles = cms.InputTag("mergedGenParticles"), # in case mergedGenParticles are created
or add an example in the test directory?